### PR TITLE
Fe feat/home page userimage and results

### PR DESCRIPTION
**Pull request by jmartin777**
# Description

1.Accepting JPEG Files: You've created a component named ImageUpload that allows users to upload JPEG images. The component consists of an input field and a drag-and-drop area.

2.State Management: states for the uploaded image (userImage), the dragging state (isDragging), and the image error message (imageError).

3.Handling Drag and Drop: You've implemented functions (handleDrag, handleDrop) to handle drag and drop events. When a user drags an image file into the drop area, you check if it's a JPEG file. If it is, you read the file using FileReader and set the base64-encoded image to the userImage state. If it's not a JPEG file, you display an error message.

4.Handling File Upload: You have a function (handleFileUpload) to handle file uploads using the input field. 

5.Display and Error Handling: You conditionally render the uploaded image or a "Upload Photo" button. If there's an error (incorrect file type), you display the error message above the upload area.

6.Rendering: The JSX structure is organized with a form that wraps the input field and the drag-and-drop area. The user can either select a file by clicking the "Upload Photo" button or by dragging and dropping a file. The image preview is displayed if a valid JPEG image is uploaded.

CSS Styling.

Closes# (issue) ImageUpload

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] Testing
- [x] Refactoring

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Components/ImageUpload/ImageUpload.js [hunk 1]
```diff
@@ -4,83 +4,78 @@ import "./ImageUpload.css"
...
+    
+    if (droppedFile && droppedFile.type.includes("jpeg")) {
+      const reader = new FileReader();
+      reader.onload = (event) => {
+        const base64 = event.target.result;
+        setUserImage(base64);
+        setImageError(null);
+        console.log("Uploaded Base64 Image:", base64);
+      };
+      reader.readAsDataURL(droppedFile);
```
Comment (andreasorensen) [inline src/Components/ImageUpload/ImageUpload.js:35]: This was so fun to implement! I'm glad we were able to figure this out without too much of a headache. Go team!

**Review by Elise-Jones — APPROVED** [review]
Good implementation for new File reader now that our post works.

File: src/Components/ImageUpload/ImageUpload.js [hunk 2]
```diff
@@ -4,83 +4,78 @@ import "./ImageUpload.css"
...
+      {userImage ? (
+        <div>
+          <img src={userImage} className="user-image" alt="mushroom" />
+          <button className="upload-button" onClick={() => setUserImage(null)}>
+            Upload New Photo
+          </button>
+        </div>
+      ) : (
+        <div className="label-photo-input">
+          <button className="upload-button" onClick={() => userFileInput.current.click()}>
```
Comment (andreasorensen) [inline src/Components/ImageUpload/ImageUpload.js:69]: 😁

**Review by andreasorensen — APPROVED** [review]
Awesome job!

**Pull request merged**